mca/identifiers.py

```diff
@@ -83,7 +103,31 @@ def get_discovery_flow_ids(self, hop, ip):
 
         return unique_flow_ids
 
-    def create_new_discovery_flow_id(self, hop, ignore=[]):
+    def create_new_discovery_flow_id(self,
+                                     hop: int,
+                                     ignore: Sequence[Sequence[int]] = None) -> tuple[int]:
+        """
+        Creates a single new flow id for a given hop, based
+        on previous knowledge of existing flow ids for the
```

Comment:
Could be more specific. We create a new flow ID that has not been sent before and is not in the list of flow IDs to ignore.